spec/javascripts/previous_winners_spec.js

```diff
@@ -0,0 +1,242 @@
+describe('PreviousWinners', function () {
```

Comment:
I think the spacing is off, or at least inconsistent. We should probably use two-space indents.